src/content/docs/get-started/team-and-account/mfa-for-kinde-access.mdx

```diff
@@ -0,0 +1,26 @@
+---
+page_id: 1338a25d-7e81-4793-8c68-57fb273c5df7
+title: Add MFA for accessing Kinde
+sidebar:
+  order: 4
+relatedArticles:
+  - f4b911df-cc23-4eb9-b3ff-56cd2bb7e378
+---
+If you want your team members to use multi-factor authentication (MFA) when they sign in to Kinde, you can contact us to request this. 
+
+We can set MFA so that it applies to everyone in your team, or only applies to certain roles, e.g. Owners within the business. 
+
+## What to ask for when you request MFA
+
+Send your MFA request to support@kinde.com, and include:
+
+- The name of each business that MFA should apply to
+- The MFA factors that you want your team to have available to them, e.g. email code, SMA code, Authenticator app.
+- Whether you want MFA to be opt-in or required
```

Comment:
@clairekinde11 - we cant allow `optional` mfa for this. I would exclude this line